packages/@uppy/status-bar/src/StatusBarUI.tsx

```diff
@@ -234,7 +240,7 @@ export default function StatusBarUI<M extends Meta, B extends Body>({
       {progressBarStateEl}
 
       <div className="uppy-StatusBar-actions">
-        {recoveredState || showUploadBtn ?
+        {showUploadBtn ?
```

Comment:
This is the main change, everything else is here in order to make this white footer collapse when nothing's inside

<img width="767" alt="image" src="https://github.com/user-attachments/assets/d07531e9-1ce9-4f95-b9da-395319a5bc70">